usr/src/uts/common/fs/zfs/zvol.c

```diff
@@ -981,9 +981,6 @@ zvol_get_done(zgd_t *zgd, int error)
 
 	zfs_range_unlock(zgd->zgd_rl);
 
-	if (error == 0 && zgd->zgd_bp)
-		zil_lwb_add_block(zgd->zgd_lwb, zgd->zgd_bp);
```

Comment:
@pfmooney I see you removed this call in both, `zfs_get_done` and `zvol_get_done`, but not `ztest_get_done`. Did you keep the `zil_lwb_add_block` call in `ztest_get_done` on purpose? I presume that's an oversight, and we also should remove that call too, but wanted to double check with you.

Reply:
Yeah, it's an oversight.  I was focused on the in-kernel bits